chapters/publication.tex

```diff
@@ -1,96 +1,171 @@
+Documents called ``notebooks'' (such as Jupyter) work similarly,
+as they also use the underlying analytical software to create the document.
+These types of dynamic documents are usually appropriate for short or informal materials
+because they tend to offer limited editability outside the base software
+and often have limited abilities to incorporate precision formatting.
+
+On the other hand, some dynamic document tools do not require
+operation of any underlying software, but simply require
+that the writer have access to the updated outputs.
+One very simple one is Dropbox Paper, a free online writing tool
+that allows linkages to files in Dropbox,
+which are then automatically updated anytime the file is replaced.
+Like the first class of tools, Dropbox Paper has very limited formatting options,
+but it is appropriate for working with collaborators who are not using statistical software.
+However, the most widely utilized software
+for dynamically managing both text and results is \LaTeX\ (pronounced ``lah-tek'').\sidenote{
+  \url{https://www.maths.tcd.ie/~dwilkins/LaTeXPrimer/GSWLaTeX.pdf}}
+  \index{\LaTeX}
+While this tool has a significant learning curve,
+its enormous flexibility in terms of operation, collaboration,
+and output formatting and styling
+makes it the primary choice for most large technical outputs today.
+In fact, \LaTeX\ operates behind-the-scenes in many of the tools listed in the first group.
+
+\subsection{Technical writing with \LaTeX}
+
+\LaTeX\ is billed as a ``document preparation system''.
+What this means is worth unpacking.
+In {\LaTeX}, instead of writing in a ``what-you-see-is-what-you-get'' mode
+as you do in Word or the equivalent,
+you write plain text interlaced with coded instructions for formatting
+(similar in concept to HTML).
+The \LaTeX\ system includes commands for simple markup
+like font styles, paragraph formatting, section headers and the like.
+But it also includes special controls for including tables and figures,
+footnotes and endnotes, complex mathematical notation, and automated bibliography preparation.
+It also allows publishers to apply global styles and templates
+to already-written material, allowing them to reformat entire documents in house styles
+with only a few keystrokes.
+In sum, \LaTeX\ enables automatically-organized documents,
+manages tables and figures dynamically,
+and (because it is written in plain text) can be version-controlled using Git.
+This is why it has become the dominant ``document preparation system'' in technical writing.
 
-The gold standard for academic writing is \LaTeX.\sidenote{\url{https://www.maths.tcd.ie/~dwilkins/LaTeXPrimer/GSWLaTeX.pdf}}
-\index{\LaTeX}
-\LaTeX\ allows automatically-organized sections like titles and bibliographies,
-imports tables and figures in a dynamic fashion,
-and can be version controlled using Git.
 Unfortunately, \LaTeX\ can be a challenge to set up and use at first,
```

Comment:
Mention that next section introduces tools that makes it easy to get started

Reply:
https://github.com/worldbank/d4di/pull/293#discussion_r369109142